e2e/Makefile

```diff
@@ -1,6 +1,6 @@
-KIND_VERSION = 0.11.1
-KUBERNETES_VERSION = 1.22.2
-CERT_MANAGER_VERSION = 1.5.3
+KIND_VERSION = 0.12.0
+KUBERNETES_VERSION = 1.23.5
```

Comment:
kind v0.12.0 allows us to use `kindest/node:v1.23.4`.
https://github.com/kubernetes-sigs/kind/releases/tag/v0.12.0

Reply:
kindest/node:v1.23.5 is pushed at Mar 25, 2022. So we can use v1.23.5.
But I will fix it to use v1.23.4.

https://hub.docker.com/layers/node/kindest/node/v1.23.5/images/sha256-a69c29d3d502635369a5fe92d8e503c09581fcd406ba6598acc5d80ff5ba81b1?context=explore

Reply:
I suspect they are preparing `kindest/node:v1.23.5` for kind v0.12.1.